src/main/java/net/datafaker/providers/videogame/Hearthstone.java

```diff
@@ -31,7 +31,7 @@ public int battlegroundsScore() {
 
     public String standardRank() {
         String rank = resolve("games.hearthstone.rank");
-        if (rank.equals("Legend")) {
+        if ("Legend".equals(rank)) {
```

Comment:
How can this return a null?

Reply:
this one probably not
however imho it's better to have constant first to avoid situations when someone uses this code as example however don't think about NPE

Reply:
yes, however here there is the opposite case:
in code we have only defined case for some specific string value and everything else...
With such approach I would consider null value as everything else